app/components/trainees/confirmation/degrees/view.rb

```diff
@@ -2,11 +2,12 @@ module Trainees
   module Confirmation
     module Degrees
       class View < GovukComponent::Base
-        attr_accessor :degrees, :trainee
+        attr_accessor :degrees, :trainee, :show_add_another_degree_button
```

Comment:
is it worth shortening this to something like: `show_degree_button`?

Reply:
I agree with @darokel. I would even go as far as creating a new method so we can put a question mark at the end e.g. `show_degree_button?`, but happy to just leave it as `show_degree_button`.

Reply:
I'm sceptical to shorten the name because there will be other options added to this component like removing a degree. You might want to add degrees but not remove degrees or visa-versa